.circleci/config.yml

```
@@ -30,3 +34,16 @@ workflows:
           matrix:
             parameters:
               tox_env: ["flake8", "mypy", "py37"]
+      - gcp-gcr/build-and-push-image:
+          # This step relies on several environment variables provided to the
+          # build job via Circle CI configuration:
+          #   GCLOUD_SERVICE_KEY
+          #   GOOGLE_PROJECT_ID
+          #   GOOGLE_COMPUTE_ZONE
```

```suggestion
      - gcp-gcr/build-and-push-image:
        context: data-eng-airflow-gcr
```

This is the new recommendation according to https://bugzilla.mozilla.org/show_bug.cgi?id=1608958 and should populate these environment variables.